src/App.tsx

```diff
@@ -15,47 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState(SortType.NONE);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const reverse = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.currentTarget.classList.toggle('is-light');
+    setIsReversed(!isReversed);
+  };
+
+  const handleReset = () => {
+    return (
+      setSortType(SortType.NONE),
+      setIsReversed(false)
+    );
```

Comment:
you don't have to return 
```suggestion
    setSortType(SortType.NONE);
    setIsReversed(false);
```